controllers/updateservice_controller.go

```diff
@@ -124,11 +124,16 @@ func (r *UpdateServiceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, err
 	}
 
+	clusterCM, _ := r.findTrustedClusterCAConfig(ctx, reqLogger, instanceCopy)
+	if errors.IsNotFound(err) {
```

Comment:
This looks weird, `err` is always nil here (otherwise we'd return on L124), is the explicit err ignore on L127 correct?